TheraLang.BLL/Services/UserManagementService.cs

```diff
@@ -46,65 +53,67 @@ public async Task<User> GetUserById(Guid id)
 
         public async Task AddUser(UserAllDto newUser)
         {
-            try
+            if (newUser != null)
             {
-                if (newUser != null)
+                var mapper = new MapperConfiguration(cfg =>
+                {
+                    cfg.CreateMap<UserAllDto, UserDetails>();
+                    cfg.CreateMap<UserAllDto, User>();
+                }).CreateMapper();
+
+                var userDetails = mapper.Map<UserAllDto, UserDetails>(newUser);
```

Comment:
Why not use ProjectTo here?

Reply:
Okay, I'll create separate pr for it, and will change it in all my services